.controlplane/entrypoint.sh

```
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Runs before the main command
+set -e
+
+echo " -- Preparing database"
+rails db:prepare
+
+echo " -- Finishing entrypoint.sh, executing '$@'"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)